javascript/javascript1/week3/Name_Removal/name.js

```diff
@@ -0,0 +1,8 @@
+const names = ['Peter', 'Ahmad', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'];
+const nameToRemove = 'Ahmad';
+
+// Write some code here
+names.splice(1,1);
```

Comment:
technically, this was supposed to be a function, where you find the index of where `nameToRemove` is. So this kind of misses the point. I'll assume you know that this was not a full effort ;)